src/base/ApplicationECSService.ts

```diff
@@ -65,6 +65,9 @@ export class ApplicationECSService extends Resource {
   ) {
     super(scope, name);
 
+    new NullProvider(this, 'ecs-null-provider');
```

Comment:
these providers are used instantiated - how are they used? is there a aws doc for these providers use?